src/configure.py

```diff
@@ -147,7 +147,7 @@ def _get_option_tuples(self, option_string):
 
         # shouldn't ever get here
         else:
-            self.error(_('unexpected option string: %s') % option_string)
+            self.error(_("unexpected option string: %s") % option_string)
```

Comment:
```suggestion
            self.error(("unexpected option string: %s") % option_string)
```
Is `_` intentionally? Static checker yell at it said that it is not defined.

Reply:
I guess so. The reference of this line is at [here](https://github.com/python/cpython/blame/43709d5d5424725ec061de2c12c3761affde69ef/Lib/argparse.py#L2321)

Reply:
I think we should remove it. As `_` is an alias from `gettext` module that we don't use at here.
([Ref](https://github.com/python/cpython/blame/43709d5d5424725ec061de2c12c3761affde69ef/Lib/argparse.py#L92))

Reply:
Agree. Let's remove it.